index.html

```diff
@@ -5,5 +5,29 @@
 </head>
```

Comment:
Your header is not correct per the assignment instructions.    Elina Pro's Page should be the title. You should  also have to meta elements.

Note: If you are using vscode, you could enter  !   and vscode will give you a template for the header.